### PR TITLE
Adding cucumber reports and slack integration 

### PR DESCRIPTION
**Pull request by kavin1711**
In this PR we have added 

1. Multiple HTML cucumber reports to the automation suite.
2. Scripts to attach screenshots of the failed scenario in the cucumber report.
3. Slack integration, where we are sending the cucumber report link to Slack once the test has failed.
4. Changed the staging URL to the production URL
5. Fixed the failed test cases.
6. Removed unwanted reports.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,25 +12,13 @@ slack-fail-post-step: &slack-fail-post-step
         branch_pattern: .+
         event: fail
         template: basic_fail_1
+        mentions: '@stdev'
         channel: C05PJHA3NTD
-        
+
```
Comment (balajikrishnannewsuk) [inline .circleci/config.yml:17]: Remove extra spaces.

File: .circleci/config.yml [hunk 2]
```diff
@@ -57,78 +45,89 @@ jobs:
...
+            {
+             "title": "Report Link",
+             "value": "'"$message"'",
+             "short": false
+            }
+            ]
+            }
+            ]
+            }' "$SLACK_WEBHOOK_URL"
+
```
Comment (balajikrishnannewsuk) [inline .circleci/config.yml:115]: Remove extra spaces.

File: .circleci/config.yml [hunk 3]
```diff
@@ -57,78 +45,89 @@ jobs:
...
-  #           echo TLS_E2E_Automation/cypress/screenshots/lhreport.html > /tmp/art-1
-  #            mkdir /tmp/artifacts
-  #     - store_artifacts:
-  #         path: /tmp/art-1
-  #         destination: artifact-file
-  #     - store_artifacts:
-  #         path: TLS_E2E_Automation/cypress/screenshots
-              
-          
+
```
Comment (balajikrishnannewsuk) [inline .circleci/config.yml:118]: Remove extra spaces.

File: .circleci/config.yml [hunk 4]
```diff
@@ -57,78 +45,89 @@ jobs:
...
 workflows:
   tests:
     jobs:
-      # - cypress-tests
-      # - test
-      - staging-health-check-tests:
+  
+      - production-health-check-tests:
           <<: *slack-fail-post-step
+
```
Comment (balajikrishnannewsuk) [inline .circleci/config.yml:126]: Remove extra spaces.

File: TLS_E2E_Automation/cypress.config.js [hunk 5]
```diff
@@ -2,17 +2,22 @@ const { defineConfig } = require( 'cypress' );
 //const ReportGenerator = require('lighthouse/lighthouse-core/report/report-generator'); 
 //const ReportGenerator = require('lighthouse/report/generator/report-generator');
 
+
```
Comment (balajikrishnannewsuk) [inline TLS_E2E_Automation/cypress.config.js:5]: Remove extra spaces.

File: TLS_E2E_Automation/cypress/support/cucumberreport.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+const report = require("multiple-cucumber-html-reporter");
+
```
Comment (balajikrishnannewsuk) [inline TLS_E2E_Automation/cypress/support/cucumberreport.js:2]: Remove extra spaces.

File: TLS_E2E_Automation/cypress/support/cucumberreport.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    title: 'TLS Healthcheck Info',
+    data: [
+        {label: 'Product', value:'The TLS'},
+        {label: 'Test', value: 'Healthcheck'},
+        {label: 'Date', value: new Date().toLocaleDateString('en-US', { day: 'numeric', month: 'short', year: 'numeric' })},
+        
+    ]
+}
+});
+
```
Comment (balajikrishnannewsuk) [inline TLS_E2E_Automation/cypress/support/cucumberreport.js:27]: Remove extra spaces.

File: TLS_E2E_Automation/cypress/support/e2e.js [hunk 8]
```diff
@@ -39,6 +40,8 @@ if (!app.document.head.querySelector('[data-hide-command-log-request]')) {
 	app.document.head.appendChild(style);
   }
 
+
```
Comment (balajikrishnannewsuk) [inline TLS_E2E_Automation/cypress/support/e2e.js:43]: Remove extra spaces.

File: TLS_E2E_Automation/cypress/support/e2e.js [hunk 9]
```diff
@@ -88,9 +91,15 @@ after( () => {
 	}
 } );
 
-afterEach( () => {
-	if(environment!='healthcheck')
-	{
-	cy.log( 'Test Completed' );
-	}
-} );
+// afterEach( () => {
```
Comment (balajikrishnannewsuk) [inline TLS_E2E_Automation/cypress/support/e2e.js:94]: Remove extra spaces.

File: TLS_E2E_Automation/cypress/support/cucumberreport.js (hunk 6 above)
Comment (kavin1711) [reply TLS_E2E_Automation/cypress/support/cucumberreport.js]: Hi @balajikrishnannewsuk, i have removed all the spaces requested.

**Review by balajikrishnannewsuk — APPROVED** [review]

**Pull request merged**